beta-3-setup.md

```diff
@@ -3762,6 +3762,10 @@ Running ansible:
     export ROUTE_53_WILDCARD_ZONE=cloudapps.example.com
     export ROUTE_53_HOST_ZONE=example.com
 
+### Bootstrap the playbook directories
+
+    ansible-playbook clone_and_setup_repo.yml
+
 ### Configuring the Hosts:
 
     ansible-playbook -i inventory/aws/hosts openshift_setup.yml
```

Comment:
On 22/05/15 00:21 -0700, Juergen Hoffmann wrote:
>>  ### Configuring the Hosts:

>>      ansible-playbook -i inventory/aws/hosts openshift_setup.yml
>
>@detiber After doing the step above this needs to be changed to `ansible-playbook -i inventory/aws/hosts/hosts openshift_setup.yml `

Ansible allows you to specify a directory for the host inventory, so we are 
relying on the dynamic inventory (inventory/aws/hosts/ec2.py) and the static 
inventory (inventory/aws/hosts/hosts) to invoke the dynamic inventory, which 
allows you to run the playbook multiple times on environment to update 
configuration or add additional nodes, etc.

I vaguely remember there being an issue with the ec2 dynamic inventory when 
there were no hosts in the environment, but I thought that had since been 
fixed.

Do you have any further information on the error you were getting with using 
inventory/aws/hosts ?